requirements/lint.txt

```diff
@@ -4,6 +4,7 @@ maxcdn
 # https://github.com/PyCQA/pylint-django/issues/117
 astroid==1.6.1
 pylint
-prospector
 pylint-django
+pylint-celery
+prospector
```

Comment:
Do you think we should pin some of these? At least to major versions?

Reply:
I think yes, but we'll need to pin minor versions. `prospector` breaking change was from 0.12.8 -> 0.12.9 :/

Reply:
Yea, our linting seems to break on a regular basis. Is there an argument for not just pinning them to a specific release? I don't feel like linting is something we need new features from very often.

Reply:
The only downside I see if that we'll need to update them manually. Should I pin them in this PR?